### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by georgehrke**
Inspired by @jancborchardt 's lightning talk earlier today

please review @raghunayyar @MariusBluem @jancborchardt

### CODE REVIEW

**Comment by georgehrke** [comment]
I also want a better screenshot, but I'll just send a pr to the screenshot repo.

**Comment by georgehrke** [comment]
better screenshot: https://github.com/nextcloud/screenshots/pull/23

File: README.md [hunk 1]
```diff
@@ -10,18 +10,22 @@
...
 
-## Maintainers:
+## Why is this so awesome?
 
- - [Georg Ehrke](https://github.com/georgehrke)
- - [Raghu Nayyar](https://github.com/raghunayyar)
- - [Thomas Citharel](https://github.com/tcitworld)
+* :rocket: **Integration with other Nextcloud apps!** Currently Contacts – more to come.
+* :globe_with_meridians: **WebCal Support!** Want to see your favorite team's matchdays in your calendar? No problem!
+* :raising_hand: **Attendees** Invite people to your events.
```
Comment (ChristophWurst) [inline README.md]: 'Attendees!'?

File: README.md [hunk 2]
```diff
@@ -10,18 +10,22 @@
...
+* :see_no_evil: **We’re not reinventing the wheel!** Based on the great [davclient.js](https://github.com/evert/davclient.js) and [ical.js](https://github.com/mozilla-comm/ical.js) libraries.
 
-## Former contributors:
- - [Bernhard Fröhler](https://github.com/codeling)
+And in the works for the [coming versions](https://github.com/nextcloud/calendar/milestones/):
+* :mag: Search for events
```
Comment (ChristophWurst) [inline README.md]: if there are existing issues, link them. e.g. https://github.com/nextcloud/calendar/issues/8

**Comment by MariusBluem** [comment]
👍

**Pull request merged**